eth/v1/validator.pb.gw.go

```diff
@@ -0,0 +1,3 @@
+// +build ignore
+
+package ignore
```

Comment:
Why do we need this? I forget...

Reply:
Part of the compilation output. I can try removing it in a later point but not in scope for this repo.